--- XQilla-2.1.1/configure.orig	2008-04-24 15:42:33.000000000 -0300
+++ XQilla-2.1.1/configure	2008-04-24 15:46:26.000000000 -0300
@@ -2668,13 +2668,13 @@
         xerces_lib="$with_xerces/lib"
         xerces_include_dir="$with_xerces/src"
         xerces_include="-I$with_xerces/src"
-elif test `ls $with_xerces/lib/libxerces-c*2[6-8].* 2>/dev/null | wc -l` -gt 0 ; then
+elif test `ls $with_xerces/lib64/libxerces-c*2[6-8].* 2>/dev/null | wc -l` -gt 0 ; then
         { echo "$as_me:$LINENO: checking for Xerces C++ version from source tree" >&5
 echo $ECHO_N "checking for Xerces C++ version from source tree... $ECHO_C" >&6; }
-        xerces_version=`ls "$with_xerces"/lib/libxerces-c*[0-9][0-9].* | tail -1 | sed 's/.*xerces-c*\([0-9]\)\([0-9]\).*/\1.\2/'`
+        xerces_version=`ls "$with_xerces"/lib64/libxerces-c*[0-9][0-9].* | tail -1 | sed 's/.*xerces-c*\([0-9]\)\([0-9]\).*/\1.\2/'`
         { echo "$as_me:$LINENO: result: $xerces_version" >&5
 echo "${ECHO_T}$xerces_version" >&6; }
-        xerces_lib="$with_xerces/lib"
+        xerces_lib="$with_xerces/lib64"
         xerces_include_dir="$with_xerces/src"
         xerces_include="-I$with_xerces/src"
 elif test `ls $with_xerces/obj/libxerces-c.* 2>/dev/null | wc -l` -gt 0 ; then
--- XQilla-2.1.1/src/config/xqilla_config.h.orig	2008-04-24 15:54:56.000000000 -0300
+++ XQilla-2.1.1/src/config/xqilla_config.h	2008-04-24 15:55:27.000000000 -0300
@@ -75,10 +75,6 @@
 #else
 
 #include "xqilla_config_unix.h"
-#ifndef int64_t
-typedef long long int64_t;
-typedef unsigned long long uint64_t;
-#endif
 
 #endif
 
